drivers/sensor/st/lps2xdf/lps2xdf.c

```diff
@@ -6,6 +6,7 @@
  * SPDX-License-Identifier: Apache-2.0
  *
  * Datasheet:
+ * https://www.st.com/resource/en/datasheet/ilps22qs.pdf
```

Comment:
You are adding a link to the datasheet in `lps2xdf.c `, but not in `lps2xdf.h´. 

What do you think of removing the set of link from one of the files?

Reply:
Yes, maybe in lps2xdf.h. 
Thanks for noticing.